docs/components_page/components/input/components_in_labels.py

```diff
@@ -0,0 +1,31 @@
+from dash import Dash, html
+import dash_bootstrap_components as dbc
+
+flights = html.Div([html.Div(className="fa fa-plane pe-1"), "Flights"])
+car = html.Div([html.Div(className="fa fa-car pe-1"), "Rental Car"])
+hotel = html.Div([html.Div(className="fa fa-hotel pe-1"), "Hotel"])
```

Comment:
These icons aren't showing up for me when I run the docs because Font Awesome stylesheet isn't actually linked.

We can add the link in `docs/templates/partials/head.html`, though I would actually have a preference for changing this example to use Bootstrap icons, since we're already linking the CSS anyway. So less for the reader of the docs to load when they visit.